### ISSUE TITLE
Test failures: "Insecure dependency in require while running with -T switch"

### ISSUE THREAD

**Issue by karenetheridge** [petdance/ack2#438]
```
cpanm (App::cpanminus) 1.7001 on perl 5.019009 built for darwin-2level
Work directory is /Users/ether/.cpanm/work/1393210028.14770
You have make /usr/bin/make
You have LWP 6.05
You have /usr/bin/tar: bsdtar 2.6.2 - libarchive 2.6.2
You have /usr/bin/unzip
Checking if you have ExtUtils::MakeMaker 6.31 ... Yes (6.90)
Checking if you have ExtUtils::Install 1.46 ... Yes (1.62)
Searching App::Ack on mirror http://mirrors.gossamer-threads.com/CPAN ...
Downloading index file http://mirrors.gossamer-threads.com/CPAN/modules/02packages.details.txt.gz ...
--> Working on App::Ack
Fetching http://mirrors.gossamer-threads.com/CPAN/authors/id/P/PE/PETDANCE/ack-2.12.tar.gz
-> OK
Unpacking ack-2.12.tar.gz
Entering ack-2.12
Checking configure dependencies from META.json
Checking if you have ExtUtils::MakeMaker 0 ... Yes (6.90)
Configuring ack-2.12
Running Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for ack
Writing MYMETA.yml and MYMETA.json
-> OK
Checking dependencies from MYMETA.json ...
Checking if you have Term::ANSIColor 1.10 ... Yes (4.02)
Checking if you have Cwd 3.00 ... Yes (3.47)
Checking if you have File::Spec 3.00 ... Yes (3.47)
Checking if you have File::Basename 1.00015 ... Yes (2.85)
Checking if you have Text::ParseWords 3.1 ... Yes (3.29)
Checking if you have File::Glob 1.00015 ... Yes (1.23)
Checking if you have Pod::Usage 1.26 ... Yes (1.63)
Checking if you have ExtUtils::MakeMaker 0 ... Yes (6.90)
Checking if you have Carp 1.04 ... Yes (1.32)
Checking if you have Getopt::Long 2.35 ... Yes (2.42)
Checking if you have File::Next 1.10 ... Yes (1.12)
Checking if you have Test::Harness 2.50 ... Yes (3.30)
Checking if you have File::Temp 0.19 ... Yes (0.2304)
Checking if you have Test::More 0.98 ... Yes (1.001002)
Building and testing ack-2.12
cp Inverse.pm blib/lib/App/Ack/Filter/Inverse.pm
cp ConfigLoader.pm blib/lib/App/Ack/ConfigLoader.pm
cp Resource.pm blib/lib/App/Ack/Resource.pm
cp Collection.pm blib/lib/App/Ack/Filter/Collection.pm
cp Is.pm blib/lib/App/Ack/Filter/Is.pm
cp Match.pm blib/lib/App/Ack/Filter/Match.pm
cp Extension.pm blib/lib/App/Ack/Filter/Extension.pm
cp Basic.pm blib/lib/App/Ack/Resource/Basic.pm
cp Ack.pm blib/lib/App/Ack.pm
cp IsGroup.pm blib/lib/App/Ack/Filter/IsGroup.pm
cp ConfigDefault.pm blib/lib/App/Ack/ConfigDefault.pm
cp ConfigFinder.pm blib/lib/App/Ack/ConfigFinder.pm
cp Resources.pm blib/lib/App/Ack/Resources.pm
cp Filter.pm blib/lib/App/Ack/Filter.pm
cp Default.pm blib/lib/App/Ack/Filter/Default.pm
cp FirstLineMatch.pm blib/lib/App/Ack/Filter/FirstLineMatch.pm
cp ExtensionGroup.pm blib/lib/App/Ack/Filter/ExtensionGroup.pm
cp ack blib/script/ack
/Users/ether/perl5/perlbrew/perls/19.9/bin/perl5.19.9 -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/ack
Manifying blib/man1/ack.0
/Users/ether/perl5/perlbrew/perls/19.9/bin/perl5.19.9 t/runtests.pl 0 0 "blib/lib" "blib/arch" t/*.t t/lib/*.t
Running tests on blib/script/ack
# Testing App::Ack 2.12, File::Next 1.12, Perl 5.019009, /Users/ether/perl5/perlbrew/perls/19.9/bin/perl5.19.9
# Using Test::More 1.001002 and Test::Harness 3.30
t/00-load.t .................. ok
t/ack-1.t .................... ok
t/ack-c.t .................... ok
t/ack-color.t ................ ok
t/ack-column.t ............... ok
t/ack-create-ackrc.t ......... ok
t/ack-dump.t ................. ok
t/ack-f.t .................... ok
t/ack-files-from.t ........... ok
t/ack-filetypes.t ............ ok
t/ack-g.t .................... ok
t/ack-group.t ................ ok
t/ack-h.t .................... ok
t/ack-help-types.t ........... ok
t/ack-help.t ................. ok
t/ack-i.t .................... ok
t/ack-ignore-dir.t ........... ok
t/ack-interactive.t .......... skipped: You need to install IO::Pty to run this test
t/ack-invalid-ackrc.t ........ skipped: Don't yet have a reliable way to ignore the Unicode complaints from Pod::Perldoc
t/ack-k.t .................... ok
t/ack-known-types.t .......... ok
t/ack-line.t ................. skipped: You need to install IO::Pty to run this test
t/ack-m.t .................... ok
t/ack-match.t ................ ok
t/ack-n.t .................... ok
t/ack-named-pipes.t .......... ok
t/ack-o.t .................... ok
t/ack-pager.t ................ skipped: You need to install IO::Pty to run this test
t/ack-passthru.t ............. ok
t/ack-print0.t ............... ok
t/ack-removed-options.t ...... ok
t/ack-s.t .................... ok
t/ack-show-types.t ........... ok
t/ack-type-del.t ............. ok
t/ack-type.t ................. ok
t/ack-v.t .................... ok
t/ack-w.t .................... ok
t/ack-x.t .................... ok
t/anchored.t ................. ok
t/asp-net-ext.t .............. ok
t/bad-ackrc-opt.t ............ ok
t/basic.t .................... ok
t/command-line-files.t ....... ok
t/config-backwards-compat.t .. ok
t/config-finder.t ............ ok
t/config-loader.t ............ ok
t/context.t .................. ok
t/default-filter.t ........... ok
t/exit-code.t ................ ok
t/ext-filter.t ............... ok
t/file-permission.t .......... ok
t/filetypes.t ................ ok
t/filter.t ................... ok
t/firstlinematch-filter.t .... ok
t/highlighting.t ............. ok
t/illegal-regex.t ............ ok
t/incomplete-last-line.t ..... ok
t/inverted-file-filter.t ..... ok
t/is-filter.t ................ ok
t/lib/00-coverage.t .......... ok
t/lib/Ack.t .................. ok
t/lib/Basic.t ................ ok
t/lib/Collection.t ........... ok
t/lib/ConfigDefault.t ........ ok
t/lib/ConfigFinder.t ......... ok
t/lib/ConfigLoader.t ......... ok
t/lib/Default.t .............. ok
t/lib/Extension.t ............ ok
t/lib/ExtensionGroup.t ....... ok
t/lib/Filter.t ............... ok
t/lib/FirstLineMatch.t ....... ok
Insecure dependency in require while running with -T switch at /Users/ether/perl5/perlbrew/perls/19.9/lib/5.19.9/base.pm line 101.
        ...propagated at /Users/ether/perl5/perlbrew/perls/19.9/lib/5.19.9/base.pm line 110.
BEGIN failed--compilation aborted at /Users/ether/.cpanm/work/1393210028.14770/ack-2.12/blib/lib/App/Ack/Filter/Inverse.pm line 5.
Compilation failed in require at t/lib/Inverse.t line 8.
BEGIN failed--compilation aborted at t/lib/Inverse.t line 8.
# Looks like your test exited with 255 before it could output anything.
t/lib/Inverse.t .............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
t/lib/Is.t ................... ok
Insecure dependency in require while running with -T switch at /Users/ether/perl5/perlbrew/perls/19.9/lib/5.19.9/base.pm line 101.
        ...propagated at /Users/ether/perl5/perlbrew/perls/19.9/lib/5.19.9/base.pm line 110.
BEGIN failed--compilation aborted at /Users/ether/.cpanm/work/1393210028.14770/ack-2.12/blib/lib/App/Ack/Filter/IsGroup.pm line 5.
Compilation failed in require at t/lib/IsGroup.t line 8.
BEGIN failed--compilation aborted at t/lib/IsGroup.t line 8.
# Looks like your test exited with 255 before it could output anything.
t/lib/IsGroup.t .............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
t/lib/Match.t ................ ok
t/lib/Resource.t ............. ok
t/lib/Resources.t ............ ok
t/longopts.t ................. ok
t/match-filter.t ............. ok
t/mutex-options.t ............ ok
t/noackrc.t .................. ok
t/noenv.t .................... ok
t/process-substitution.t ..... ok
t/r-lang-ext.t ............... ok
t/resource-iterator.t ........ ok
t/zero.t ..................... ok

Test Summary Report
-------------------
t/lib/Inverse.t            (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
t/lib/IsGroup.t            (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
Files=86, Tests=908, 66 wallclock secs ( 0.66 usr  0.27 sys + 55.88 cusr  9.67 csys = 66.48 CPU)
Result: FAIL
Failed 2/86 test programs. 0/908 subtests failed.
make: *** [test_classic] Error 255
-> FAIL Installing App::Ack failed. See /Users/ether/.cpanm/work/1393210028.14770/build.log for details. Retry with --force to force install it.
```

**Comment by petdance** [comment]
@karenetheridge Is this still happening in 2.16?

**Comment by karenetheridge** [comment]
It looks like this got fixed somewhere between 2.12 and 2.14. thanks!
